static/app/components/events/interfaces/spans/traceErrorList.tsx

```diff
@@ -13,32 +13,55 @@ import {
 import {ParsedTraceType, SpanType} from './types';
 
 interface TraceErrorListProps {
-  errors: (TraceError | TracePerformanceIssue)[];
+  errors: TraceError[];
   trace: ParsedTraceType;
+  performanceIssues?: TracePerformanceIssue[];
 }
 
-function TraceErrorList({trace, errors}: TraceErrorListProps) {
+function TraceErrorList({trace, errors, performanceIssues}: TraceErrorListProps) {
   return (
     <List symbol="bullet" data-test-id="trace-error-list">
-      {flatten(
-        Object.entries(groupBy(errors, 'span')).map(([spanId, spanErrors]) => {
-          return Object.entries(groupBy(spanErrors, 'level')).map(
-            ([level, spanLevelErrors]) => (
-              <ListItem key={`${spanId}-${level}`}>
-                {tct('[errors] [link]', {
-                  errors: tn(
-                    '%s %s error in ',
-                    '%s %s errors in ',
-                    spanLevelErrors.length,
-                    level === 'error' ? '' : level // Prevents awkward "error errors" copy if the level is "error"
-                  ),
-                  link: findSpanById(trace, spanId).op,
-                })}
-              </ListItem>
-            )
-          );
-        })
-      )}
+      <Fragment>
+        {flatten(
+          Object.entries(groupBy(errors, 'span')).map(([spanId, spanErrors]) => {
+            return Object.entries(groupBy(spanErrors, 'level')).map(
+              ([level, spanLevelErrors]) => (
+                <ListItem key={`${spanId}-${level}`}>
+                  {tct('[errors] [link]', {
+                    errors: tn(
+                      '%s %s error in ',
+                      '%s %s errors in ',
+                      spanLevelErrors.length,
+                      level === 'error' ? '' : level // Prevents awkward "error errors" copy if the level is "error"
+                    ),
+                    link: findSpanById(trace, spanId).op,
+                  })}
+                </ListItem>
+              )
+            );
+          })
+        )}
+        {flatten(
+          Object.entries(groupBy(performanceIssues, 'span')).map(
+            ([spanId, spanErrors]) => {
+              return Object.entries(groupBy(spanErrors, 'level')).map(
+                ([level, spanLevelErrors]) => (
+                  <ListItem key={`${spanId}-${level}`}>
```

Comment:
Is this supposed to `groupBy` something else? I'm just wondering because I think this will create duplicate keys and I'm not 100% sure it's necessary because we're not rendering any information about levels